net-analyzer/linkchecker/linkchecker-10.0.0_pre20200729.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
```

Comment:
```suggestion
PYTHON_COMPAT=( python3_{6..9} )
```

Reply:
Missing support in pyxdg: https://bugs.gentoo.org/735746

Reply:
Isn't this resolved?